test/WebSites/ApiExplorerWebSite/project.json

```diff
@@ -3,18 +3,23 @@
     "web": "ApiExplorerWebSite"
   },
   "compilationOptions": {
-    "emitEntryPoint": true
+    "emitEntryPoint": true,
+    "preserveCompilationContext": true
   },
   "dependencies": {
     "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*",
     "Microsoft.AspNetCore.Mvc": "1.0.0-*",
     "Microsoft.AspNetCore.Mvc.Formatters.Xml": "1.0.0-*",
     "Microsoft.AspNetCore.Mvc.TestConfiguration": "1.0.0",
-    "Microsoft.AspNetCore.StaticFiles": "1.0.0-*",
-    "Microsoft.NETCore.Platforms": "1.0.1-*"
+    "Microsoft.AspNetCore.StaticFiles": "1.0.0-*"
   },
   "frameworks": {
-    "dnx451": {},
-    "dnxcore50": {}
+    "net451": {},
+    "dnxcore50": {
+      "imports": "portable-net451+win8",
```

Comment:
We consume a portable class library (one of the CodeAnalysis packages) and consequently have to do this.